lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -321,6 +322,14 @@ int main(int argc, char* argv[]) {
     }
   });
 
+  // Start off a thread to periodically poll stats from Pipelined
+  // every fixed interval of time
+  auto periodic_stats_requester = std::make_shared<magma::PollStats>();
+  std::thread periodic_stats_requester_thread([&]() {
+    periodic_stats_requester->start_loop(local_enforcer, 5);
```

Comment:
How did we decide on this 5 value? Is this the default polling config? Will we be loading this from a config eventually?

Reply:
Yes we will load from a config eventually, I just left it in as standard value

Reply:
Can you add a comment to indicate this?